tests/db_tests/test_api.py

```diff
@@ -58,13 +59,17 @@ def try_to_mount_device():
 
     # mock output of actual mounting command
     def mount(path):
-        return "Mounted"
+        return "Mounted at /dummy/path"
 
     monkeypatch.setattr(fit_collector, "_mount_device_using_gio", mount)
+    monkeypatch.setattr(fit_collector, "_mount_device_using_pmount", mount)
 
     # mock output of _find_device_type
     def _find_device_type(bus, dev):
-        return ("MTP", "/dev/bus/usb/001/002")
+        if mock_dev == "MTP":
+            return ("MTP", "/dev/bus/usb/001/002")
+        elif mock_dev == "BLOCK":
+            return ("BLOCK", "/dev/sda")
```

Comment:
The test `test_mount_device__success` could be considered a smoke test since it only checks the the api call returns a 200 status code. Unfortunately testing the mounting of devices is obviously not really doable in a automated fashion, which is why most of the mounting functionality is mocked away. One could argue that this test isn't really helpful at all, I guess.

The referenced lines here are just used to mock/overwrite the actual implementation of `_find_device_type`, thus this test does not really test the actual implementation. Of course there is no harm in doing this, but I honestly doubt that there is any benefit in doing this. It could even make us feel like we are testing the implementation, which creates a false sense of security.